lib/player.js

```diff
@@ -173,6 +174,18 @@ goog.requireType('shaka.routing.Payload');
  * @exportDoc
  */
 
+/**
+ * @event shaka.Player.MediaQualityChangedEvent
+ * @description Fired when the media quality changes at the playhead.
+ * That may be caused by an adapatation change or a DASH period transition.
+ * Separate events are emitted for audio and video contentTypes.
```

Comment:
Probably worth mentioning that (at least for the moment) this only works for DASH.

Reply:
done